shell/common/crash_keys.cc

```diff
@@ -205,6 +207,10 @@ void SetCrashKeyForGinWrappable(gin::WrapperInfo* info) {
   else if (info == &electron::api::DesktopCapturer::kWrapperInfo)
     crash_location = "DesktopCapturer";
 #endif
+#if !defined(OS_WIN)
+  else if (info == &electron::api::Tray::kWrapperInfo)
```

Comment:
hold up, we still build `Tray` on windows though right?

Reply:
We do, yes - I think the header file for tray_icon might be missing an import when it's linked into the crash keys file. This is the specific error that the build is throwing on main now: https://ci.appveyor.com/project/electron-bot/electron-woa-release/build/1.0.1087#L53457

I tried to find the missing import and then opted for this solution as a fix to get main up and running again, but also happy to find the import so we can actually check for a pointer matching the Tray's gin wrappable, either in this PR or in a fast follow 🙂

Reply:
Let's fix it for real in this PR. Sounds like it might be as straightforward as IWYUing `electron_api_tray.h`.

Reply:
Doing the import fix now, please hold

Reply:
Update: We took a stab at this and it ended up being more complicated that expected, will make an accompanying issue with notes to continue to fix it, since we should find a long-term fix